### ISSUE TITLE
http specification tests ConnectionManagementIT, TransferCodingsIT failing with read... instead of closed

### ISSUE THREAD

**Issue by cmebarrow** [k3po/k3po#260]
The failures indicate data was read unexpectedly before the closed event came in:
```
Failed tests:
  ConnectionManagementIT.serverMustCloseConnectionAfterResponseWithConnectionClose Specified behavior did not match expected:<...write "\r\n"

close
[closed]

> but was:<...write "\r\n"

close
[read ...]

>
  TransferCodingsIT.shouldProcessRequestTransferEncodingChunked Specified behavior did not match expected:<... 200 OK\r\n"

close
[closed]

accept http://loca...> but was:<... 200 OK\r\n"

close
[read ...]

accept http://loca...>
```

**Comment by cmebarrow** [comment]
For the first failure at the issue is that script `specification/http/src/main/scripts/org/kaazing/specification/http/rfc7230/connection.management/server.must.close.connection.after.response.with.connection.close/response.rpt` fails to read all the data written by the request script. It is a miracle this test ever passed!

**Comment by cmebarrow** [comment]
I have a fix ready

**Comment by cmebarrow** [comment]
Fixed in PR #263
